docs/source/install.rst

```diff
@@ -118,7 +110,7 @@ These optional dependencies and their minimum supported versions are listed belo
 +---------------+----------+--------------------------------------------------------------+
 |   sqlalchemy  |          |            Writing and reading from SQL databases            |
 +---------------+----------+--------------------------------------------------------------+
-| cytoolz/toolz | >=0.8.2  | Utility functions for iterators, functions, and dictionaries |
+|    cytoolz    | >=0.8.2  | Utility functions for iterators, functions, and dictionaries |
```

Comment:
I think this should be reworded to explain that toolz is a mandatory dependency but it can be transparently replaced with cytoolz.